test/d-b-n_smoke/arrays.html

```diff
@@ -248,6 +248,26 @@ <h4>
                         }, 100);
                     });
                 });
+
+                describe('Undesired but expected. Polymer does not reflect deleting props', function () {
```

Comment:
We will definitelly need this in release notes, even though behavior was changed long time ago with P2 upgrade